proposals/0167-app-services.md

```diff
@@ -1006,11 +1018,11 @@ The HMI will be responsible for the actual selection and activation of app servi
 </function>
 
 <function name="AppServiceActivation" messagetype="response">
-  <param name="serviceId" type="String" mandatory="true" >
+  <param name="serviceID" type="String" mandatory="true" >
     <description>The ID of the service that was requested to have an activation event take place</description>
   </param>
-  <param name="serviceType" type="Common.ServiceType" mandatory="true">
-    <description>The service type the service that was requested to have an activation event take place</description>
+  <param name="serviceType" type="String" mandatory="true">
+    <description>The service type the service that was requested to have an activation event take place. See AppServiceType</description>
```

Comment:
Update to:
    <description>**The service type **of** the service** that was requested to have an activation event take place. See AppServiceType</description>